packages/components/scroll-area/tests/scroll-area.test.ts

```diff
@@ -14,6 +31,149 @@ import ChromaticDynamicContent from '../src/story/ChromaticDynamicContent.vue'
 enableAutoUnmount(afterEach)
 
 describe('OkuScrollArea', () => {
+  it('OkuScrollArea renders correctly', () => {
+    const wrapper = mount(OkuScrollArea)
```

Comment:
I got `TypeError: Cannot read properties of undefined (reading 'devtoolsRawSetupState')` here